sql/updates/latest-dev.sql

```diff
@@ -102,3 +102,15 @@ INSERT INTO _timescaledb_config.bgw_policy_drop_chunks (
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_config.bgw_policy_drop_chunks', '');
 DROP TABLE _timescaledb_config.bgw_policy_drop_chunks_tmp;
 GRANT SELECT ON _timescaledb_config.bgw_policy_drop_chunks TO PUBLIC;
+
+DROP VIEW IF EXISTS timescaledb_information.compressed_hypertable_stats;
+DROP VIEW IF EXISTS timescaledb_information.compressed_chunk_stats;
+
+-- all existing compressed chunks have NULL value for the new columns
+ALTER TABLE IF EXISTS _timescaledb_catalog.compression_chunk_size ADD COLUMN IF NOT EXISTS total_uncompressed_rows BIGINT;
+ALTER TABLE IF EXISTS _timescaledb_catalog.compression_chunk_size ADD COLUMN IF NOT EXISTS total_compressed_rows BIGINT;
+
+--rewrite catalog table to not break catalog scans on tables with missingval optimization
+CLUSTER  _timescaledb_catalog.compression_chunk_size USING compression_chunk_size_pkey;
+ALTER TABLE _timescaledb_catalog.compression_chunk_size SET WITHOUT CLUSTER;
```

Comment:
I don't understand why this is necessary. Can you elaborate?

Reply:
PG optimizes the case where  we alter a table and add a new column. Old data in the table  will not have this new column value after the ALTER. This forces it to write the data out with the newly added column.